ui/locales/app.fr.json

```
@@ -348,7 +329,7 @@
   "topographic_profile_header": "Profil de l'altitude ",
   "tracking_agree_label": "Consentement à la acquisition de données (facultatif)",
   "tracking_data_acquisition_header": "Acquisition de données",
-  "tracking_data_acquisition_text": "Nous utilisons les outils d’analyse Google Analytics et Sentry afin d’adapter au mieux notre site web à vos besoins. Votre comportement sur le site est enregistré de manière anonyme. Aucune donnée personnelle n’est transmise ou stockée. Si vous n’acceptez pas cet état de fait, vous pouvez empêcher la collecte de données par des outils d’analyse et continuer à utiliser ce site web sans restriction. Vous trouverez plus d’informations à ce sujet sur notre page <a target='_blank' href='https://www.swissgeol.ch/datenschutz-fr'>Informations juridiques.</a>",
+  "tracking_data_acquisition_text": "Nous utilisons l'outil d’analyse Google Analytics afin d’adapter au mieux notre site web à vos besoins. Votre comportement sur le site est enregistré de manière anonyme. Aucune donnée personnelle n’est transmise ou stockée. Si vous n’acceptez pas cet état de fait, vous pouvez empêcher la collecte de données par des outils d’analyse et continuer à utiliser ce site web sans restriction. <b> Vous trouverez plus d’informations à ce sujet sur notre page <a target='_blank' href='https://www.swissgeol.ch/datenschutz-fr'>Informations juridiques.</a>",
```

`br` anstelle von `b`.
```suggestion
  "tracking_data_acquisition_text": "Nous utilisons l'outil d’analyse Google Analytics afin d’adapter au mieux notre site web à vos besoins. Votre comportement sur le site est enregistré de manière anonyme. Aucune donnée personnelle n’est transmise ou stockée. Si vous n’acceptez pas cet état de fait, vous pouvez empêcher la collecte de données par des outils d’analyse et continuer à utiliser ce site web sans restriction. <br> Vous trouverez plus d’informations à ce sujet sur notre page <a target='_blank' href='https://www.swissgeol.ch/datenschutz-fr'>Informations juridiques.</a>",
```